lib/App/DuckPAN/Fathead.pm

```diff
@@ -105,10 +105,10 @@ sub _search_output {
 
 	my $result = $self->_db_lookup($query);
 
-	if ($result && $result->{type} eq "R") {
+	while ($result && $result->{type} eq 'R') {
```

Comment:
@moollaza I've run into a couple of situations where it wants to run more than once; but yeah, probably not very common.